clinica/iotools/converters/aibl_to_bids/aibl_utils.py

```diff
@@ -36,58 +36,65 @@ def find_T1_folder(subdirectory, path_to_T1_1):
         "MPRAGE_ADNI_confirmed_REPEAT",
         "MPRAGE_ADNI_conf_REPEAT",
     }
+
+    path = None
+
     for j in path_to_convert:
-        path = []
         # if conditions which checks if the subfolder contain a T1 image
         if j == subdirectory:
             path = os.path.join(path_to_T1_1, subdirectory)
             return path
-    if path == []:
+
+    if not path:
         return "NaN"  # there are no more folders which could contain T1 images
 
 
 def find_T1_folder_nodata(subdirectory, path_to_T1_1):
     """
-
     This method checks if the subdirectory contains a T1 image, and it
     returns the path. This method differs from the find_T1_folder since for
-    these folders the exame_date is not present in the clinical excel file
-    and we will not check if the exame_date corresponds to the date stored
-    in the path to the image, but they will be converted anyway
+    these folders the exam_date is not present in the clinical Excel file,
+    and it will not check if the exam_date corresponds to the date stored
+    in the path to the image, but they will be converted anyway.
 
-     :param subdirectory: name of the folder
+    :param subdirectory: name of the folder
 
-     :return: previous path to arrive to the T1 image
+    :return: previous path to arrive to the T1 image
     """
     import os
 
-    path_to_convert = {"MPRAGESAGISOp2ND", "MPRAGE_SAG_ISO_p2_ND", "MPRAGE_SAG_ISO_p2"}
+    path_to_convert = {
+        "MPRAGESAGISOp2ND",
+        "MPRAGE_SAG_ISO_p2_ND",
+        "MPRAGE_SAG_ISO_p2",
+    }
+
+    path = None
+
     for j in path_to_convert:
-        path = []
```

Comment:
This pattern is a copy-paste from `find_T1_folder`. Could be worth factorizing it out. WDYT?